### PR TITLE
Blaze: Update dark mode color for intro view and campaign creation form

### LINKED ISSUE #11735: Update view colors for dark mode
Screenshot 2024-01-19 at 11 40 31

### PR DESCRIPTION
**Pull request by itsmeichigo**
Closes: #11735 

## Description

This PR updates the colors on the intro view and campaign creation form to match the design (mBhyoV7OPhFnOugxLy6SMS-fi-2506-59795).

## Testing instructions

- Log in to a store that's eligible for Blaze and doesn't have any campaign created yet.
- Tap on the Promote button on the Blaze section on the dashboard screen.
- Notice that the intro screen has colors matching the design in both light and dark modes.
- Tap Create Your Campaign, notice that the creation form is pushed with the tab bar hidden.
- Notice that the colors on the creation form match the design in both light and dark modes.

## Screenshots

\ | Light mode | Dark mode 
--- | --- | ---
Intro view |  | 
Creation form |  |  

---
- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

File: WooCommerce/Classes/ViewRelated/Blaze/BlazeIntro/BlazeCreateCampaignIntroView.swift [hunk 1]
```diff
@@ -52,11 +52,11 @@ struct BlazeCreateCampaignIntroView: View {
                         Image(uiImage: .blaze)
                             .resizable()
                             .renderingMode(.template)
-                            .foregroundColor(Color(.brand))
+                            .foregroundColor(Color(.accent))
```
Comment (selanthiraiyan) [inline WooCommerce/Classes/ViewRelated/Blaze/BlazeIntro/BlazeCreateCampaignIntroView.swift:55]: Thanks for fixing this.

File: WooCommerce/Classes/ViewRelated/Blaze/CampaignCreation/BlazeCampaignCreationForm.swift [hunk 2]
```diff
@@ -182,11 +188,12 @@ private extension BlazeCampaignCreationForm {
                 // Image
                 Image(uiImage: viewModel.image?.image ?? .blazeProductPlaceholder)
                     .resizable()
-                    .aspectRatio(contentMode: .fill)
+                    .aspectRatio(contentMode: .fit)
                     .cornerRadius(Layout.cornerRadius)
 
                 // Tagline
                 Text(viewModel.isLoadingAISuggestions ? "Placeholder tagline" : viewModel.tagline)
+                    .foregroundStyle(.secondary)
```
Comment (selanthiraiyan) [inline WooCommerce/Classes/ViewRelated/Blaze/CampaignCreation/BlazeCampaignCreationForm.swift:202]: For my learning: What will this modifier do? When should be use this for a Text?

File: WooCommerce/Classes/ViewRelated/Blaze/CampaignCreation/BlazeCampaignCreationForm.swift [hunk 3]
```diff
@@ -182,11 +188,12 @@ private extension BlazeCampaignCreationForm {
                 // Image
                 Image(uiImage: viewModel.image?.image ?? .blazeProductPlaceholder)
                     .resizable()
-                    .aspectRatio(contentMode: .fill)
+                    .aspectRatio(contentMode: .fit)
```
Comment (selanthiraiyan) [inline WooCommerce/Classes/ViewRelated/Blaze/CampaignCreation/BlazeCampaignCreationForm.swift:197]: Should we update the `BlazeEditAdView` image's aspect ratio as well?

**Review by selanthiraiyan — APPROVED** [review]
Thanks for working on this. 🚀

File: WooCommerce/Classes/ViewRelated/Blaze/CampaignCreation/BlazeCampaignCreationForm.swift (hunk 2 above)
Comment (itsmeichigo) [reply WooCommerce/Classes/ViewRelated/Blaze/CampaignCreation/BlazeCampaignCreationForm.swift:202]: Xcode says `foregroundColor` will be deprecated in a future OS version and we should switch to this to reduce the code change we'll need to do in the future. This modifier can apply the same style to a group of text, images, and shapes. You can check out the [documentation](https://developer.apple.com/documentation/swiftui/view/foregroundstyle(_:)) and [this article](https://www.hackingwithswift.com/quick-start/swiftui/how-to-provide-visual-structure-using-foreground-styles) for more information.

File: WooCommerce/Classes/ViewRelated/Blaze/CampaignCreation/BlazeCampaignCreationForm.swift (hunk 3 above)
Comment (itsmeichigo) [reply WooCommerce/Classes/ViewRelated/Blaze/CampaignCreation/BlazeCampaignCreationForm.swift:197]: Done in 0b423c97100bdcd4eeeee1046f135f69450513c3.

**Pull request merged**